api-spec.md

```diff
@@ -135,7 +135,7 @@ This Link should look like:
 
 Unless otherwise noted by **Path-only**, these filters are passed as query string parameters, form parameters, or JSON 
 entity fields.  For filters that represent a set of values, query and form parameters should use comma-separated 
-string values and JSON entity attributes should use JSON Arrays. 
+string values (with no outer brackets \[ or \]) and JSON entity attributes should use JSON Arrays. 
```

Comment:
I'm confused. So it's comma separated values, but "JSON entity attributes should use JSON Arrays"? Does that conflict? What does "JSON entity attributes should use JSON Arrays" actually mean?

Reply:
I'm 90% sure that this means:
- If you are using query string parameters (GET request to `/search/`) you should do bbox=10,10,20,20
- If you are using JSON requests (POST to `/search/`) then you should do "bbox": [10,10,20,20]

But obviously, that is not clear. If my explanation makes sense then I can expand this description out. It occurs to me we should also just remove the 'form parameters', and perhaps put it back in if we do https://github.com/radiantearth/stac-api-spec/issues/4

Reply:
Oh, I see. Let's better fix that, too, but I'm approving anyway for now as the main issue has been solved.